src/Dropdown/Dropdown.scss

```diff
@@ -1,2 +1,10 @@
 @import "variables";
 @import "~bootstrap/scss/dropdown";
+
+.dropdown-toggle::after {
+    border: 0;
+    content: "\f107";
+    font-family: "fontawesome";
```

Comment:
Paragon we should avoid fontawesome icon font dependencies. Could you bake in this svg: 
https://github.com/edx/paragon/blob/master/icons/svg/expand_more.svg

Similar to how this one form errors is done?
- https://github.com/edx/brand-edx.org/blob/master/paragon/_overrides.scss#L180
- https://github.com/edx/paragon/blob/master/src/Form/_variables.scss#L130

Reply:
@abutterworth how can we define the variable to fill the svg? because the text color comes from the button and it is defined in a mixin.

Reply:
`fill="currentColor"` in an svg will inherit the text color of the html. https://css-tricks.com/cascading-svg-fill-color/

Reply:
@abutterworth fill="currentColor" doesn't work in css :https://stackoverflow.com/a/20515297 only works in inline svg, so maybe Do we have to modify the react files?

Reply:
Hmm, I see. Let's avoid wading into react for now. Is it possible to get this done with a css only solution the looks like our chevron svg? https://codepen.io/egf/pen/jzylw

Reply:
It works, I updated the new design image with the result.